tests/avalon_test_framework/conftest.py

```diff
@@ -1,15 +1,36 @@
 import utility.logger as plogger
 import env
+import toml
 import pytest
 import json
 import os
 import sys
 import logging
 import config.config as pconfig
+from http_client.http_jrpc_client import HttpJrpcClient
 
 logger = logging.getLogger(__name__)
 sys.path.append(os.getcwd())
 
+# Config File Path
```

Comment:
Should we put all the code till line 32 in setup_config function itself? rather than outside?